src/System.Private.ServiceModel/tools/scripts/OpenFirewallPorts.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off 
+setlocal
+
+REM 
+REM Start Open all ports WCF test services use
+REM
+
```

Comment:
I think you need to do some kind of elevated prompt check too, as all the netsh commands will fail if you run this while not elevated. A trick I've used before is to run the command "net session" which requires admin rights to execute properly. 

somthing like: 

```
net session > nul
if not %ERRORLEVEL% equ 0 (
    echo this script must be run elevated.  
    exit /b 1
)